### DISCUSSION TITLE
Does the Invalid Movies (Series) Auto Cleaner delete the files on the disk or just from the 'arr?

### DISCUSSION THREAD

**Discussion by DataLoreQ** [RandomNinjaAtk/arr-scripts#255] [Q&A] [answered]
As the title says, looking to find out if the Invalid Movies (Series) Auto cleaner deletes the files on the disk or just from the 'arr itself, such as Radarr.

I have noticed, on occasion, a movie gets deleted from TMDB, but the movie was actually released.  I had a few of them on my last installation (reinstalling everything right now), but didn't have it activated.  So, I would would like to know the answer before I enable it if at all possible!

Thank You In Advance!

BTW, LOVE these scripts!

**Comment by RandomNinjaAtk** [comment, marked as answer]
Going by memory, I think it both wipes the data if it exists and removes it from the application.

To some this might be undesirable, but the goal of these scripts is to automate it, and normally this would be an active warning requiring user intervention.

**Comment by DataLoreQ** [comment]
Thank you for that!  That was what I was leaning towards and answer, but I like to make sure!
